### PR TITLE
Add fluentd document

### PR DESCRIPTION
**Pull request by repeatedly**
For https://github.com/docker-library/official-images/pull/5303

### CODE REVIEW

**Comment by repeatedly** [comment]
Hmm... travis-ci error is unclear for me. Newline is not allowed in one paragraph?

**Comment by yosifkit** [comment]
Yeah, paragraphs should be a single line and paragraphs are separated by a single empty line.

**Comment by repeatedly** [comment]
Okay, fixed it.

File: fluentd/content.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+```bash
+$ docker run -p 24224:24224 -p 24224:24224/udp -u fluent -v /path/to/dir:/fluentd/log %%IMAGE%%
+2019-01-16 11:49:55 +0000 [info]: parsing config file is succeeded path="/fluentd/etc/fluent.conf"
+...
+2019-01-16 11:58:27 +0000 [info]: #0 [input1] listening port port=24224 bind="0.0.0.0"
+2019-01-16 11:58:27 +0000 [info]: #0 fluentd worker is now running worker = 0
+```
+
+Docker's official images support only v1.3.3 or later. If you want to know more details, check [fluentd-docker-image README](https://github.com/fluent/fluentd-docker-image/blob/master/README.md).
```
Comment (yosifkit) [inline fluentd/content.md]: Probably just need to update this to be 1.4.2 or later :wink:.

Comment (repeatedly) [reply fluentd/content.md]: Thanks! Just updated.

**Pull request merged**